### ISSUE TITLE
403 Forbidden Error within Jobs via Unpublish & Duplicate Buttons

### ISSUE THREAD

**Issue by Mindy-Moz** [mozmeao/snippets-service#1272]
Hi @glogiotatidis 

I'm having trouble un-publishing jobs individually or duplicating after cancelling, once I click the button, it leads me to a 403 Forbidden error.

Currently the only way I can un-publish is through the snippet, viewing all scheduled jobs, selecting all and then canceling via the drop down action. Let me know if the platform is undergoing any changes. 

Thanks,
Mindy

**Comment by glogiotatidis** [comment]
Hi Mindy, I believe you're experiencing #1150, sorry for your troubles. Will target to take care of this by EOW

**Comment by glogiotatidis** [comment]
Hi Mindy! Call you please test and let me know if you still experience problems? If so please give my the Job ids that you have trouble iwth

**Comment by Mindy-Moz** [comment]
Hi Giorgos,

The error is still occurring, I think this is happening for all jobs, when
I've scheduled it and for some reason need to unpublish within the actual
job, it will lead to a 403 error.

Here is a snippet I just pushed to publish: 22167
<https://snippets-admin.mozilla.org/admin/base/job/22167/change/> but now I
am unable to unpublish via the "Unpublish Button (Cancel)".

The Job:
[image: Screen Shot 2019-12-18 at 11.57.21 AM.png]

The Error:
[image: Screen Shot 2019-12-18 at 11.57.32 AM.png]


The second part to the 403 error is the in-ability to duplicate "cancelled"
jobs. (I may do this because of the need to postpone runtime to a few hours
or the next day). I find this is affecting potentially all cancelled jobs.

22036
<https://snippets-admin.mozilla.org/admin/base/job/22036/change/?_changelist_filters=q%3D22036>
This is a sample of a cancelled job, which I don't need to run anymore. But
if I click on "Duplicate" it leads me to the 403 error.
Currently, I just rebuild the job from scratch.
I am able to Duplicate if the job has not been scheduled before.

The Cancelled Job:
[image: Screen Shot 2019-12-18 at 11.58.39 AM.png]

The Error:
[image: Screen Shot 2019-12-18 at 11.58.48 AM.png]
Thanks,
Mindy
